app/contact/page.tsx

```diff
@@ -1,5 +1,212 @@
+"use client";
```

Comment:
While I think the whole page shouldn't be client side and only the form should be, it seems that this page is only going to contain the form right? Since the footer is probably in a layout.tsx file.